### DISCUSSION TITLE
Which are the ports that a manager node use to comunicate with a forward node?

### DISCUSSION THREAD

**Discussion by Angelozinna96** [Security-Onion-Solutions/securityonion#4663] [Unsupported Versions]
Hi everyone,
I'm planning to deploy a forward node outside the network where the other components of the distributed deployment are, so I have to set all the ports used for communication from and to a manager node.
Looking into https://docs.securityonion.net/en/2.3/firewall.html, I've seen which are the ports for a manager node  (22, 3142, 5000, 8086, 4505, 4506, 5644, 443, 8080).

I would like to know which are the forward node ports that are used by a manager node to work properly (if any). 
Thanks in advance!

**Comment by weslambert** [comment]
Please see the following: https://docs.securityonion.net/en/latest/firewall.html#node-communication
